js/htsget/htsgetBamReader.js

```diff
@@ -55,6 +55,7 @@ class HtsgetBamReader extends HtsgetReader {
 
         // BAM decoding
         const ba = BGZip.unbgzf(compressedData.buffer)
+        this.header = BamUtils.decodeBamHeader(ba, this.genome)
```

Comment:
`header` needed to be decoded again due to difference in `header.size`.  If relying on the first assignment (line 45), the offset will be wrong.

Reply:
The change is harmless, but I would like to understand it.    Why would header.size differ? Shouldn't the header be exactly the same?

Reply:
The header isn't exactly the same.  We are using `samtools` in our htsget implementation and it outputs the command used in the PG header fields, which differs between fetching header only vs full data.

Here's an example diff:
![header-diff](https://github.com/user-attachments/assets/da7bad48-7864-41f9-9659-c9cf9c4fd98b)

Reply:
OK, that's reasonable.  Odd that no one has  encountered this before but there's no real evidence htsget is being used in igv.js until now.